### PR TITLE
Support for Managed SANs

### PR DESCRIPTION
**Pull request by marikrg**

### CODE REVIEW

File: hpOneView/resources/fc_sans/managed_sans.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+        return self._client.get(id_or_uri=id_or_uri)
+
+    def update(self, id_or_uri, data, timeout=-1):
+        """
+        Updates a Managed SAN.
+
+        It's possible:
+        - Refresh the Managed SAN.
+        - Updating the Managed SAN's publicAttributes.
+        - Updating the Managed SAN's policy.
```
Comment (balestrinc) [inline hpOneView/resources/fc_sans/managed_sans.py]: For it be displayed as a list you should add an indentation

File: hpOneView/resources/fc_sans/managed_sans.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+            query:
+                A general query string to narrow the list of resources returned.
+                The default is no query - all resources are returned.
+            sort:
+                The sort order of the returned data set. By default, the sort order is based
+                on create time, with the oldest entry first.
+
+        Returns:
+            list: A list of Managed SANs
+
```
Comment (balestrinc) [inline hpOneView/resources/fc_sans/managed_sans.py]: Remove this blank line

File: hpOneView/resources/fc_sans/managed_sans.py (hunk 1 above)
Comment (marikrg) [inline hpOneView/resources/fc_sans/managed_sans.py]: It's good to know that.

File: hpOneView/resources/fc_sans/managed_sans.py (hunk 2 above)
Comment (marikrg) [inline hpOneView/resources/fc_sans/managed_sans.py]: Done!

**Pull request merged**